src/datanode/src/datanode.rs

```diff
@@ -49,6 +50,7 @@ pub struct DatanodeOptions {
     pub storage: ObjectStoreConfig,
     pub enable_memory_catalog: bool,
     pub mode: Mode,
+    pub tls: Option<Arc<TlsOption>>,
```

Comment:
I think we don't need this `TlsOption` in datanode because only frontend serves user's traffic.

Reply:
if internal traffic will not use secure connection in the.future，datanode can remove this tls option，otherwise datanode can set tls option to none.

Reply:
Since we don't use it in datanode, I think it's safe to put `TlsOption` into `servers` module.

Even if in future we have secure internal communication between datanode/frontend/meta, we don't use this configuration item for internal security. (Consider how kubernetes manages its internal ssl and ingress ssl)

Reply:
got it.